tools/goctl/api/swagger/swagger.go

```diff
@@ -259,12 +259,24 @@ func wrapCodeMsgProps(ctx Context, properties spec.SchemaProps, atDoc apiSpec.At
 	if !ctx.WrapCodeMsg {
 		return properties
 	}
+
+	type WrapCodeMsgMapping struct {
+		Code string `json:"code"`
+		Data string `json:"data"`
+		Msg  string `json:"msg"`
+	}
+
+	var wrapCodeMsgMapping WrapCodeMsgMapping
+	if err := json.Unmarshal([]byte(ctx.WrapCodeMsgMapping), &wrapCodeMsgMapping); err != nil {
+		return properties
+	}
+
 	globalCodeDesc := ctx.BizCodeEnumDescription
 	methodCodeDesc := getStringFromKVOrDefault(atDoc.Properties, propertyKeyBizCodeEnumDescription, globalCodeDesc)
 	return spec.SchemaProps{
 		Type: []string{swaggerTypeObject},
 		Properties: spec.SchemaProperties{
-			"code": {
+			wrapCodeMsgMapping.Code: {
```

Comment:
Pls ignore this operation if the value of `wrapCodeMsgMapping.Code` is empty, and the same operation with`wrapCodeMsgMapping.Data`, `wrapCodeMsgMapping.Msg` too.